postreise/analyze/generation/capacity_value.py

```diff
@@ -57,3 +61,39 @@ def calculate_net_load_peak(scenario, resources, hours=100):
     net_demand = total_demand - resource_generation
     top_hours = net_demand.sort_values(ascending=False).head(hours).index
     return resource_generation[top_hours].mean()
+
+
+def get_capacity_by_resources(scenario, area, resources, area_type=None):
+    """Get total capacity value of certain resources in the specific area of a
+    scenario.
+
+    :param powersimdata.scenario.scenario.Scenario scenario: scenario instance
+    :param str area: one of: *loadzone*, *state*, *state abbreviation*,
```

Comment:
the second colon is weird. I would remove it, i.e., `one of *'loadzone'*, *'state'*, *'state abbreviation'*, *'interconnect'*, *'all'*